site/en/r2/guide/using_gpu.ipynb

```diff
@@ -70,7 +70,7 @@
         "\n",
         "Note: Use the `tf.test.is_gpu_available` function to confirm that TensorFlow is using the GPU.\n",
         "\n",
-        "The simplest way to run on multiple GPUs, on one or many machines, is using [Distribution Strategies](../distribute_strategy.ipynb)\n",
+        "The simplest way to run on multiple GPUs, on one or many machines, is using [Distribution Strategies](../guide/distribute_strategy.ipynb)\n",
```

Comment:
SInce its in the same directory, it works with [distribute_strategy.ipynb] too :)

Reply:
Its throwing 404 on the main website and on GitHub too

Reply:
:o thanks for the tip!

Reply:
Right. Changing it to [Distribution Strategies](distribute_strategy.ipynb) will work too.